updating DIP for W10.16 opto-couplers

easyw opened the pull request:
models are generated by parametric scripts, WRL with KiCAD suggested
materials

https://github.com/easyw/kicad-3d-models-in-freecad/tree/master/cadquery/FCAD_script_generator

discussion related at

https://forum.kicad.info/t/contributing-to-update-of-vrml-librarys-to-step/5049/
![housings_dip_opto-copuplers](https://cloud.githubusercontent.com/assets/3032347/23697439/148c963a-03e8-11e7-8b59-806153580677.png)

SchrodingersGat commented:
@jkriege2 can you please check over this one? :)

jkriege2 commented:
I will check these this evening ...

jkriege2 commented:
Hi!

for the high-pincount devices the body width was correct. We will only need these (i.e. small body, large pitch) for 4-pin, 6-pin and 8-pin.
I'll PR the missing footprints for 6- and 8-pins soon.

So could you generate 6- and 8-pin and remove 22- and 24-pin?

Thanks & best,
JAN

easyw commented:
removed 22,24 pins, added 4, 6 and 8 pins with a more realistic case
![housing_dip_w1016 3](https://cloud.githubusercontent.com/assets/3032347/23717196/1aaaa566-0433-11e7-9819-e1c41a878454.png)

jkriege2 commented:
please remove the MSOP, SSOP, ... packages fromt his PR!

jkriege2 commented:
see https://github.com/KiCad/Housings_DIP.pretty/pull/5 for a matching PR for footprints!

easyw commented:
- removed next PR for SSOP
- packages already fine (built in line with the family)
![housing_dip 2](https://cloud.githubusercontent.com/assets/3032347/23724723/1869756e-044e-11e7-91bd-65e21087f998.png)

jkriege2 commented:
Great work!

Thanks,
JAN
